### PR TITLE
Use new tower template for continuous deploy

### PR DESCRIPTION
**Pull request by maxkadel**
In order to enhance security, we are using a template that only runs on the main branch of princeton ansible and deploys only the main branch of orangelight for continuous deployment.

### CODE REVIEW

**Review by christinach — APPROVED** [review]
Thanks @maxkadel

File: .circleci/config.yml
Comment (sandbergja) [inline .circleci/config.yml:206]:
```suggestion
          command: "curl -k \"https://ansible-tower.princeton.edu/api/v2/job_templates/57/launch/\" --header \"Content-Type: application/json\" --header \"Authorization: Bearer $TOWER_TOKEN\" -d '{\"credential_passwords\":{},\"extra_vars\":{\"repo_name\":\"orangelight\",\"runtime_env\":\"staging\",\"slack_alerts_channel\":\"ansible-alerts\"}}'"          
```
Since main is now harcoded, can we stop passing it?

**Review by sandbergja — COMMENTED** [review]
Thanks, @maxkadel 🦄 , just a small cleanup question (can wait until after we merge and validate this definitely).

**Review by sandbergja — APPROVED** [review]

**Pull request merged**
